doc/install.rst

```diff
@@ -35,9 +35,17 @@ If you want to use Argbash locally, you have to download the software package an
    * Supply ``INSTALL_COMPLETION=yes`` as an installation argument to install bash completion for ``argbash`` to ``$(SYSCONFDIR)/bash_completion.d``.
      The default ``SYSCONFDIR`` is ``/etc``, but you may override it in the same way as you can override the ``PREFIX`` variable.
 
+   * To be able to let Argbash :ref:`generate manpages <manpage_output>`, |rst2man|_ needs to be installed on your system. It's part of the Python package |docutils|_ which can be installed on Debian/Ubuntu using ``sudo apt install python3-docutils``.
```

Comment:
You can use ``:ref:`docutils <https://pypi.org/project/docutils/>` `` or `` `docutils <https://pypi.org/project/docutils/>`_`` to get rid of those aliases and separate hyperlinks, see e.g. https://docutils.sourceforge.io/docs/user/rst/quickref.html#external-hyperlink-targets

Reply:
Ok, done :)